kong/plugins/http-log/handler.lua

```diff
@@ -14,10 +14,16 @@ local HTTPS = "https"
 -- @param `parsed_url` contains the host details
 -- @param `message`  Message to be logged
 -- @return `body` http payload
```

Comment:
I think these comments need to be updated, they do not match the code anymore (not caused by this PR, but would be nice to fix that in one go). And some others in the code too.